polkadot-js/alphanet-types.json

```diff
@@ -0,0 +1,26 @@
+{
+  "AccountId": "EthereumAccountId",
+  "Address": "AccountId",
+  "Balance": "u128",
+  "RefCount": "u8",
+  // mapping the lookup
```

Comment:
Comments are technically not valid json, I think.

That being said, if tools that use this data can handle it, I'm fine keeping it.